wcfsetup/install/files/lib/system/form/builder/field/label/LabelFormField.class.php

```diff
@@ -36,6 +38,8 @@ final class LabelFormField extends AbstractFormField implements IObjectTypeFormN
      */
     protected $labelGroup;
 
+    protected LabelPickerGroup $labelPickerGroup;
```

Comment:
I don't quite understand the addition of the `labelPickerGroup` here, since the whole logic just needs to turn it back into the singular `LabelPicker` anyways. Is this in anticipation of some future change or does `LabelPicker` itself not provide the necessary logic?

Reply:
The form field itself is super awkward because it should have never been a label selection itself and rather a collection of label groups. The intended design is to use `LabelPickerGroup` to delegate all tasks but since the form field focuses on a single field I have to do some code gymnastics to make it work.